lib/network/api/resource/operation.go

```diff
@@ -0,0 +1,42 @@
+package resource
+
+import (
+	"strings"
+
+	"boscoin.io/sebak/lib/block"
+
+	"github.com/nvellon/hal"
+)
+
+type Operation struct {
+	bo *block.BlockOperation
+}
+
+func NewOperation(bo *block.BlockOperation) *Operation {
+	o := &Operation{
+		bo: bo,
+	}
+	return o
+}
+
+func (o Operation) GetMap() hal.Entry {
+	return hal.Entry{
+		"id":      o.bo.Hash,
+		"hash":    o.bo.Hash,
+		"funder":  o.bo.Source,
+		"account": o.bo.Target,
+		"type":    o.bo.Type,
+		"amount":  o.bo.Amount.String(),
+	}
+}
+
+func (o Operation) Resource() *hal.Resource {
+
```

Comment:
:) Please remove blank line.

Reply:
Yep! haha

Reply:
@anarcher https://github.com/bosnet/sebak/pull/363/files#diff-07e238e3bf2f5ad137b6f8e21c6a9f06R32 has still empty blank line :)

Reply:
Thanks @spikeekips! :-)   I fix it.  I don't know why my linter (gometalinter) doesn't check it ㅠㅠ